homeassistant/components/minecraft_server/sensor.py

```diff
@@ -29,6 +33,65 @@
 from .entity import MinecraftServerEntity
 
 
+@dataclass
+class MinecraftServerEntityDescriptionMixin:
+    """Mixin values for Minecraft Server entities."""
+
+    value_fn: Callable[[MinecraftServerData | None], StateType]
+
+
+@dataclass
+class MinecraftServerSensorEntityDescription(
+    SensorEntityDescription, MinecraftServerEntityDescriptionMixin
+):
+    """Class describing Minecraft Server sensor entities."""
+
+
+SENSOR_DESCRIPTIONS = [
+    MinecraftServerSensorEntityDescription(
+        key=KEY_VERSION,
+        translation_key=KEY_VERSION,
+        icon=ICON_VERSION,
+        value_fn=lambda data: data.version if data else None,
```

Comment:
Let's remove the inline ifs by setting available to super().available and data is not None

Reply:
I added the inline ifs, because the dataclass ("data" in the lambdas) can be None itself, therefore something (I think pylint) was complaining about that "None" doesn't have attributes ("version" in this case).
I saw this method of inline ifs in the elgato integration.

Reply:
Let me propose something:

Change the current available property to `self._server.online and self._server.data is not None` I mean I am now assuming that `self._server.online` is false when data is None. When available is False, `native_value` isn't called.

Then change the async_update to a property native_value where you just 
```python
@property
def native_value(self) -> None:
    assert self._server.data
    return self.entity_description.value_fn()
```
and also don't forget to change the type of `value_fn` to remove the | None

Reply:
I can't remove the async_update function, as I have not moved to the data update coordinator yet. But you pointed to the right direction. I think I had a fault in the value_fn type. Please check the latest changes, the code now works without the inline ifs in the lambdas.